packages/lib/gpc/src/gpcTypes.ts

```diff
@@ -85,23 +85,14 @@ export type GPCProofEntryConfig = {
   equalsEntry?: PODEntryIdentifier;
 
   /**
-   * Indicates list(s) in which this entry must lie. An entry value may lie in
-   * one or more membership lists.
-   *
-   * Note that the underlying GPC expects a one-to-one correspondence between
-   * comparison values and lists, i.e. it will only check one value per
-   * list. Thus, if multiple entry values are constrained to be members of some
-   * fixed list, this list will be duplicated for each of these checks,
-   * increasing the circuit size requirements. This should be taken into account
-   * when estimating circuit sizes.
+   * Indicates a list in which this entry must lie.
```

Comment:
Somewhere in the public docs there should be a discussion of the limits and flexibility of list membership.  That could belong here, or on the input type (with references between them), or it could be duplicated.
I think the docs need to point out:
- 1 list per entry/tuple, either member or nonmember not both
- Each list can be reused, but will be duplicated in the circuit (so users don't think they're saving anything by reusing a list)
- Recommend that cases which call for multiple lists can always be addressed by constructing a combined list (intersection, union, subtraction), which isn't more expensive due to the way duplicates work above.